# :star: Phylogenetic Trees [2025]

<br>

![](https://img.shields.io/badge/MAXIMUM%20LIKELIHOOD%20-Randomized%20Axelerated%20Maximum%20Likelihood%20[RAxML]-green?style=for-the-badge)

> Edler, D., Klein, J., Antonelli, A. & Silvestro, D. (2020) raxmlGUI 2.0: A graphical interface and toolkit for phylogenetic analyses using RAxML. *Methods in Ecology and Evolution*, 12, 373–377.

> Stamatakis, A. (2014) RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. *Bioinformatics*, 30 (9), 1312-1313.

***- RAxML-GUI (2.0.14) with one single run, the GTRGAMMAI model and 1000 thorough bootstrap replicates.***

<img src="https://github.com/Rohit-Rannavre/Phylogenetic-Trees/blob/main/Trees/RAxML-GUI.png" width="600" height="400">

***- RAxML-GUI (2.0.14) with one single run, the GTRGAMMA model as selected by PartitionFinder (2.1.1) for codon-based partitions and 1000 thorough bootstrap replicates.***

<img src="https://github.com/Rohit-Rannavre/Phylogenetic-Trees/blob/main/Trees/RAxML-GUI-2.png" width="600" height="400">

![](https://img.shields.io/badge/MAXIMUM%20LIKELIHOOD%20-IQ--TREE-eb3471?style=for-the-badge)

> Hoang, D. T., Chernomor, O., Haeseler, A., Minh, B. Q. & Vinh, L. S. (2018) UFBoot2: improving the ultrafast bootstrap approximation. *Molecular Biology and Evolution*, 35 (2), 518–522.

> Kalyaanamoorthy, S., Minh, B. Q., Wong, T. K. F., von Haeseler, A. & Jermiin, L. S. (2017) ModelFinder: fast model selection for accurate phylogenetic estimates. *Nature Methods*, 14, 587-589.

> Minh, B. Q., Schmidt, H. A., Chernomor, O., Schrempf, D., Woodhams, M. D., von Haeseler, A. & Lanfear, R. (2020) IQ-TREE 2: new models and efficient methods for phylogenetic inference in the genomic era. *Molecular Biology and Evolution*, 37 (5), 1530-1534.

***- IQ-TREE-CLI (2.4.0) with automated model selection, based on BIC, via ModelFinder (TPM3uFG4) and 1000 ultrafast bootstrap replicates.***

<img src="https://github.com/Rohit-Rannavre/Phylogenetic-Trees/blob/main/Trees/IQTREE.png" width="600" height="400">

***- IQ-TREE-CLI (2.4.0) with the models GTR+G, HKY+G and HKY as selected by PartitionFinder (2.1.1) for codon-based partitions and 1000 ultrafast bootstrap replicates.***

<img src="https://github.com/Rohit-Rannavre/Phylogenetic-Trees/blob/main/Trees/IQTREE-2.png" width="600" height="400">

![](https://img.shields.io/badge/BAYESIAN%20INFERENCE%20-MRBAYES-ebe534?style=for-the-badge)

> Ronquist, F., Teslenko, M., van der Mark, P., Ayres, D. L., Darling, A., Höhna, S., Larget, B., Liu, L., Suchard, M. A. & Huelsenbeck, J. P. (2012) MrBayes 3.2: efficient bayesian phylogenetic inference and model selection across a large model space. *Systematic Biology*, 61 (3), 539-542.

***- MrBayes-CLI (3.2.7) with 2 runs, 4 chains, the GTRGAMMAI model, 1000000 generations and 25% burn-in.***

![](https://img.shields.io/badge/BAYESIAN%20INFERENCE%20-Bayesian%20Evolutionary%20Analysis%20Sampling%20Trees%20[BEAST]-d924ed?style=for-the-badge)

<br>

### ***Let's connect:*** 
[![LinkedIn](https://img.shields.io/badge/linkedin-%230077B5.svg?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/rohit-rannavre) 
[![Gmail](https://img.shields.io/badge/Gmail-D14836?style=for-the-badge&logo=gmail&logoColor=white)](mailto:rohit.rannavre@gmail.com)
