# :star: Phylogenetic Trees [2025]

<br>

![](https://img.shields.io/badge/MAXIMUM%20LIKELIHOOD%20-Randomized%20Axelerated%20Maximum%20Likelihood%20[RAxML]-green?style=for-the-badge)

> Edler, D., Klein, J., Antonelli, A. & Silvestro, D. (2020) raxmlGUI 2.0: A graphical interface and toolkit for phylogenetic analyses using RAxML. *Methods in Ecology and Evolution*, 12, 373â€“377.

> Stamatakis, A. (2014) RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. *Bioinformatics*, 30 (9), 1312-1313.

***- RAxML-CLI (8.2.10) with the GTRCAT model, based on one single run and 1000 thorough bootstrap replicates.***

<img src="https://github.com/Rohit-Rannavre/Phylogenetic-Trees/blob/main/Trees/1.RAxML-CLI.png" width="600" height="400">

***- RAxML-GUI (2.0.14) with the GTRGAMMAI model and 1000 rapid bootstrap replicates.***

<img src="https://github.com/Rohit-Rannavre/Phylogenetic-Trees/blob/main/Trees/2.RAxML-GUI.png" width="600" height="400">

![](https://img.shields.io/badge/MAXIMUM%20LIKELIHOOD%20-Phylogenetic%20Estimation%20Using%20Maximum%20Likelihood%20[PhyML]-54c7b9?style=for-the-badge)

> Guindon, S. & Gascuel, O. (2003) PhyML: A simple, fast and accurate algorithm to estimate large phylogenies by maximum likelihood. *Systematic Biology*, 52 (5), 696-704.

> Guindon, S., Dufayard, J.F., Lefort, V., Anisimova, M., Hordijk, W. & Gascuel, O. (2010) New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0. *Systematic Biology*, 59 (3), 307-321.

> Lefort, V., Longueville, J-E. & Gascuel, O. (2017) SMS: Smart Model Selection in PhyML. *Molecular Biology and Evolution*, 34 (9), 2422-2424.

***- PhyML-CLI (3.1) with the GTR model and 1000 bootstrap replicates.***

<img src="https://github.com/Rohit-Rannavre/Phylogenetic-Trees/blob/main/Trees/3.PhyML-CLI.png" width="600" height="400">

***- PhyML-Server (3.0) with automated model selection via SMS (GTRG) and 1000 bootstrap replicates.***

<img src="https://github.com/Rohit-Rannavre/Phylogenetic-Trees/blob/main/Trees/4.PhyML-Server.png" width="600" height="400">

![](https://img.shields.io/badge/MAXIMUM%20LIKELIHOOD%20-IQ--TREE-eb3471?style=for-the-badge)

> Kalyaanamoorthy, S., Minh, B. Q., Wong, T. K. F., von Haeseler, A. & Jermiin, L. S. (2017) ModelFinder: fast model selection for accurate phylogenetic estimates. *Nature Methods*, 14, 587-589.

> Minh, B. Q., Schmidt, H. A., Chernomor, O., Schrempf, D., Woodhams, M. D., von Haeseler, A. & Lanfear, R. (2020) IQ-TREE 2: new models and efficient methods for phylogenetic inference in the genomic era. *Molecular Biology and Evolution*, 37 (5), 1530-1534.

> Trifinopoulos, J., Nguyen, L. T., von Haeseler, A. & Minh, B. Q. (2016) W-IQ-TREE: a fast online phylogenetic tool for maximum likelihood analysis. *Nucleic Acids Research*, 44 (1), 232-235.

***- IQ-TREE-CLI (2.4.0) with automated model selection via ModelFinder (TPM3uFG4) and 1000 ultrafast bootstrap replicates.***

<img src="https://github.com/Rohit-Rannavre/Phylogenetic-Trees/blob/main/Trees/5.IQTREE-CLI.png" width="600" height="400">

***- IQ-TREE-Server with automated model selection via ModelFinder (TPM3FG4) and 100 standard bootstrap replicates.***

<img src="https://github.com/Rohit-Rannavre/Phylogenetic-Trees/blob/main/Trees/6.IQTREE-Server.png" width="600" height="400">

<br>

### ***Let's connect:*** 
[![LinkedIn](https://img.shields.io/badge/linkedin-%230077B5.svg?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/rohit-rannavre) 
[![Gmail](https://img.shields.io/badge/Gmail-D14836?style=for-the-badge&logo=gmail&logoColor=white)](mailto:rohit.rannavre@gmail.com)
